go/vt/vtctl/cells_aliases.go

```diff
@@ -64,53 +65,53 @@ func init() {
 	})
 }
 
-func commandAddCellsAlias(ctx context.Context, wr *wrangler.Wrangler, subFlags *flag.FlagSet, args []string) error {
-	cellsString := subFlags.String("cells", "", "The list of cell names that are members of this alias.")
+func commandAddCellsAlias(ctx context.Context, wr *wrangler.Wrangler, subFlags *pflag.FlagSet, args []string) error {
+	cells := subFlags.StringSlice("cells", nil, "The list of cell names that are members of this alias.")
 	if err := subFlags.Parse(args); err != nil {
 		return err
 	}
 	if subFlags.NArg() != 1 {
 		return fmt.Errorf("the <alias> argument is required for the AddCellsAlias command")
 	}
 
-	cells := strings.Split(*cellsString, ",")
-	for i, cell := range cells {
-		cells[i] = strings.TrimSpace(cell)
+	// cells := strings.Split(*cellsString, ",")
```

Comment:
Did you intend to leave the commented line? There is one more below.

Reply:
ohhh weird, no i didn't!